routing.rst

```diff
@@ -49,22 +60,43 @@ any PHP class stored in the ``src/Controller/`` directory.
 Suppose you want to define a route for the ``/blog`` URL in your application. To
 do so, create a :doc:`controller class </controller>` like the following::
 
-    // src/Controller/BlogController.php
-    namespace App\Controller;
+.. configuration-block::
 
-    use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
-    use Symfony\Component\Routing\Annotation\Route;
+    .. code-block:: php-attributes
```

Comment:
Er should decided if we use attributes as the first and default tab or as the second tab and keep annotations the default and first tab for now.

We can then configure doctor-rst to stay consistent.

WDYT @symfony/team-symfony-docs ?

Reply:
Since attributes require PHP 8 I suggest to have it as the second tab and display a format by default that can be used with PHP 7 too.

Reply:
I will update `DOCtor` to expect `php-attributes` at second 👍

Reply:
All right, I will swap the tabs.

Reply:
Done in https://github.com/OskarStark/doctor-rst/pull/761, please rebase again. It should fail now, until you swap the tabs 👍